src/GeneralStateTestsFiller/stExtCodeHash/extCodeHashPrecompilesFiller.yml

```diff
@@ -0,0 +1,160 @@
+# EXTCODEHASH of precompiles addresses
+---
+extCodeHashPrecompiles:
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x20000'
+    currentGasLimit: "1000000"
+    currentNumber: "1"
+    currentTimestamp: "1000"
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+  pre:
+    095e7baea6a6c7c4c2dfeb977efac326af552d87:
+      balance: '1000000000000000000'
+      code: |
+        {
+          (CALL 150000 0xaaaaaaaa00000000000000000000000000000001 0 0 0 0 32)
+          (RETURNDATACOPY 0 0 32)
+          [[0]] (MLOAD 0)
+          (CALL 150000 0xaaaaaaaa00000000000000000000000000000002 0 0 0 0 32)
+          (RETURNDATACOPY 0 0 32)
+          [[1]] (MLOAD 0)
+          (CALL 150000 0xaaaaaaaa00000000000000000000000000000003 0 0 0 0 32)
+          (RETURNDATACOPY 0 0 32)
+          [[2]] (MLOAD 0)
+          (CALL 150000 0xaaaaaaaa00000000000000000000000000000004 0 0 0 0 32)
+          (RETURNDATACOPY 0 0 32)
+          [[3]] (MLOAD 0)
+          (CALL 150000 0xaaaaaaaa00000000000000000000000000000005 0 0 0 0 32)
+          (RETURNDATACOPY 0 0 32)
+          [[4]] (MLOAD 0)
+          (CALL 150000 0xaaaaaaaa00000000000000000000000000000006 0 0 0 0 32)
+          (RETURNDATACOPY 0 0 32)
+          [[5]] (MLOAD 0)
+          (CALL 150000 0xaaaaaaaa00000000000000000000000000000007 0 0 0 0 32)
+          (RETURNDATACOPY 0 0 32)
+          [[6]] (MLOAD 0)
+          (CALL 150000 0xaaaaaaaa00000000000000000000000000000008 0 0 0 0 32)
+          (RETURNDATACOPY 0 0 32)
+          [[7]] (MLOAD 0)
+        }
+      nonce: '0'
+      storage: { # initialize storage values to 0x01
+        0: "0x01",
+        1: "0x01",
+        2: "0x01",
+        3: "0x01",
+        4: "0x01",
+        5: "0x01",
+        6: "0x01",
+        7: "0x01"
+        }
+    aaaaaaaa00000000000000000000000000000001:
+      balance: '1000000000000000000'
+      code: |
+        {
+          (MSTORE 0 (EXTCODEHASH 0x0000000000000000000000000000000000000001))
```

Comment:
extcodehash / extcodsize  are mixed here (in this test) inconsistent. 
you could also pass the contract that you want to call as DATA for the call.

```
(MSTORE 0 0x1122...32) (CALL 50 account 0 0 32 0 0)
```

and load that data in account with `(CALLDATALOAD 0)`